pkgs/games/quake3/ioquake/default.nix

```diff
@@ -56,6 +57,12 @@ stdenv.mkDerivation {
     mkdir -p $out/baseq3
   '';
 
+  postInstall = ''
+    mkdir $out/bin
+    makeWrapper $out/ioquake3.x86_64 $out/bin/ioquake3
+    makeWrapper $out/ioq3ded.x86_64 $out/bin/ioq3ded
```

Comment:
Should be `.arm64` on aarch64-linux.

Reply:
Thank you. Fixed.

When I realized supported platforms are all `lib.platforms.linux` it became obvious this was something to defer to the package's installer. We now simply specify `COPYBINDIR` to `installFlags`.